scripts/activate-venv.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+if ! (return 0 2>/dev/null); then
+    echo -e "\nThis script must be sourced, see below\n"
+    echo -e "Usage: 'source scripts/activate-venv.sh'\n"
+    return 1
```

Comment:
If we're inside this `then` branch, that happened because `return` wouldn't be valid outside a function (because the script is being executed rather than sourced, the only time `return` is valid outside a function in `bash`). So, this is the only time in a `source`-only script when you *should* use `exit`—because it's handling the case where a `source`-only script is being executed.

Because of this, when executed, there's some extra noise and errors from the errant call to `return`.

```suggestion
    exit 1
```

Reply:
Good point, fixed